native/cocos/audio/android/PcmAudioService.cpp

```diff
@@ -113,7 +125,7 @@ bool PcmAudioService::init(AudioMixerController *controller, int numChannels, in
     const SLInterfaceID ids[] = {
         SL_IID_PLAY,
         SL_IID_VOLUME,
-        SL_IID_ANDROIDSIMPLEBUFFERQUEUE,
+        IDD_BUFFER_QUEUE,
```

Comment:
Does that means Openharmony does not support standard flags as `SL_IID_ANDROIDSIMPLEBUFFERQUEUE`?

Reply:
Yes, this is unique to Android. Openharmony uses other